content/articles/dhcp-server-router-configuration/index.md

```diff
@@ -0,0 +1,203 @@
+R2(config)#ip dhcp excluded-address 192.168.2.1 192.168.2.10
+R2(config)#ip dhcp excluded-address 192.168.2.50
+R2(config)#exit
+```
+### DHCP Server Verification.
+To confirm that our DHCP server is configured on the two routers, `show running-config | section DHCP` command is used. The output is as shown below, which confirms the routers can now perform the functionalities of assigning IP addresses dynamically to the network clients.
+
+
+```bash
+R2#show running-config | section dhcp
+ip dhcp excluded-address 192.168.2.100
+ip dhcp excluded-address 192.168.2.1 192.168.2.10
+ip dhcp excluded-address 192.168.2.50
+ip dhcp pool POOL-B
+ network 192.168.2.0 255.255.255.0
+ default-router 192.168.2.100
+ dns-server 192.168.2.50
+```
+>**NOTE: The dhcp server can be disabled using `no service dhcp` command and re-enabled using `service dhcp` command in the global configuration mode.**
+### DHCP Server Router Testing
+Once we have confirmed the routers have now configured to act as DHCP servers, we can test their functionalities by trying to assign Ip addresses to the clients. From our network, we have 4 PCs acting as the clients. Before the configurations, the router configured as the DHCP server could not assign the PCs the address dynamically, the DHCP failed as shown.
+
+![Failed DHCP Configuration](/engineering-education/dhcp-server-router-configuration/fail-dhcp.jpg)
+
+After the configuration, we'll try again to assign each client an IP address by selecting the DHCP option on the IP configuration interface window. The DHCP works successfully. This shows our routers are now DHCP enabled.
+
+![Success DHCP Configuration](/engineering-education/dhcp-server-router-configuration/success-dhcp.jpg)
+
+We can also try to test the communication between two clients on the same subnet using the `ping` command. Our network, PC5 can communicate with PC4 using the assigned IP addresses as shown.
+
+![Ping Network Configuration](/engineering-education/dhcp-server-router-configuration/ping-dhcp.jpg)
+
+>**NOTE: From the IP addresses assigned it is evident that the DHCP server assigned addresses from .11 excluding the IP addresses from .1 to .10 and .50 which were among the excluded addresses.**
+
+### Conclusion
+As we have seen, network routers can save the organization the cost of buying dedicated servers by providing the DHCP server functionality when configured properly. This information is crucial and will equip the learner with the knowledge of how to configure the routers to eliminate the burden of statically assigning IP addresses.
+
+
+To summarize:
+- We learned what a DHCP server is.
+- We explored how to use a router to act as a DHCP server.
+- We configured a network, verified and configured the routers to perform the DHCP server functions.
+- We learned how to enable and disable the DHCP server on the network.
+
+One can find more information about  DHCP-router configuration [here](https://contenthub.netacad.com/srwe/7.2.1).
+
+Happy coding.
```

Comment:
Consider adding a couple of references or additional material for the reader to get more value even after reading your tutorial.